projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -248,6 +248,7 @@ export interface GridServiceType {
     getChildGrid?(path: IPathSegment[]): GridType;
     // XXX: Fix type
     unsetChildRowIsland?(rowIsland: any): void;
+    endEditAll?(): void;
```

Comment:
Is this a leftover? It's not part of the row, I find it only in hgridAPI service.